src/mixins/I13nMixin.js

```diff
@@ -108,8 +109,16 @@ var I13nMixin = {
         self._i13nNode.setDOMNode(ReactDOM.findDOMNode(self));
 
         // enable viewport checking if enabled
-        if (self._getReactI13n().isViewportEnabled()) {
-            self.subscribeViewportEvents();
+        if (reactI13n.isViewportEnabled()) {
+            var options = {};
+            if ( reactI13n.getScrollableContainerId &&
+                 reactI13n.getScrollableContainerId() ) {
+                if (reactI13n.getScrollableContainerDOMNode &&
+                    reactI13n.getScrollableContainerDOMNode()) {
+                    options.target = reactI13n.getScrollableContainerDOMNode();
```

Comment:
seems like we don't need to check `reactI13n.getScrollableContainerId()`, just do `reactI13n.getScrollableContainerDOMNode()` and decide if we want to apply the container setting